core/src/main/resources/lib/form/confirm.js

```diff
@@ -15,6 +15,20 @@
         return errorMessage;
       }
     }
+    
+    function isIgnoringConfirm(element){
+      if(element.hasClassName('ignore-confirm')){
+        return true;
+      }
+      // to allow sub-section of the form to ignore confirm
+      // especially useful for "pure" JavaScript area
+      var ignorePanel = element.up('.ignore-confirm-panel');
```

Comment:
I wonder whether a more general name would be preferable here, as this effectively becomes API. Confirmation panel is a specific effect of a 'dirty' or modified form.

Reply:
`no-dirty-trigger` ?
`stay-pristine` ? (mix of Angular form validation and Britney Spears) _(joke)_